discovery-provider/src/queries/get_playlist_tracks.py

```diff
@@ -83,11 +83,18 @@ def get_playlist_tracks(session, args):
         playlists_map = {}
         for playlist in playlists:
             playlist_id = playlist["playlist_id"]
+            playlist_is_public = not playlist["is_private"]
             playlists_map[playlist_id] = []
             for track_id_dict in playlist["playlist_contents"]["track_ids"]:
                 track_id = track_id_dict["track"]
                 track = track_ids_map[track_id]
-                playlists_map[playlist_id].append(track)
+                track_is_public = not track["is_unlisted"]
```

Comment:
i wonder what happens in the case of tracks that have been made is_delete true /is_available false?

Reply:
i think we would want to hide any tracks with is_available false (delisted tracks)

Reply:
https://github.com/AudiusProject/audius-client/blob/edb31425c30baeaa9b5e81e05dc49f68c55f43f3/packages/web/src/pages/track-page/TrackPageProvider.tsx#L442

Reply:
we could filter it at the db level too but then the client may never see those tracks, is there a case we'd want that?